shogun-lib/src/main/java/de/terrestris/shogun/lib/model/Topic.java

```diff
@@ -43,6 +55,7 @@
     @OneToOne
     @Fetch(FetchMode.JOIN)
     @Audited(targetAuditMode = RelationTargetAuditMode.NOT_AUDITED)
+    @Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
```

Comment:
not sure why used explicitly on a single field here while enabled globally on top

Reply:
This is because collections are not cached by default and need to be explicitly marked as cacheable.